algorithms/scaling/observers.py

```diff
@@ -42,8 +43,13 @@
     make_image_range_table,
 )
 from dials.algorithms.scaling.scale_and_filter import make_scaling_filtering_plots
+from dials.algorithms.scaling.scaling_library import (
+    merging_stats_from_scaled_array,
+    DialsMergingStatisticsError,
```

Comment:
Discussion topic: are custom specific exceptions a useful thing?

Reply:
Useful in this context I believe, as merging statistics code can throw a number of different exceptions which can be difficult to predict.